com.sweethome3d.Desktop.appdata.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2019 Bastien Nocera <hadess@hadess.net> -->
+<component type="desktop">
+  <id>com.sweethome3d.Desktop</id>
```

Comment:
I think there were some issues when an app used .Desktop on their app-id but I don't remember if that was fixed. @TingPing do you know what was the status on this?

Reply:
Should be fine, there's a bunch of special detection for this (telegram is the app you're thinking of)